iina/OpenURLWindowController.swift

```diff
@@ -74,7 +74,8 @@ class OpenURLWindowController: NSWindowController, NSTextFieldDelegate, NSContro
     guard !urlField.stringValue.isEmpty else { return (nil, false) }
     let username = usernameField.stringValue
     let password = passwordField.stringValue
-    guard var urlValue = urlField.stringValue.addingPercentEncoding(withAllowedCharacters: .urlAllowed) else {
+    let trimmedUrlString = urlField.stringValue.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
```

Comment:
Just a question: does `urlField.stringValue.trimmingCharacters(in: .whitespacesAndNewlines)` not work here?

Reply:
I'm sure it does, I just used the explicit type to hopefully get quicker autosuggestions and then forgot to shorten it. I'll shorten it.

Reply:
Thanks!

Reply:
Done